packages/shylu/shylu_node/tacho/src/impl/Tacho_SupernodeInfo.hpp

```diff
@@ -20,6 +20,13 @@ Sandia National Laboratories, Albuquerque, NM, USA
 #define __TACHO_SUPERNODE_INFO_HPP__
 
 #include "Tacho_Util.hpp"
+#if defined(KOKKOS_ENABLE_CUDA)
```

Comment:
I'm not sure this is sufficient to ensure cusparse is available? Would be helpful to add more conditions to the check though I'm not sure what is sufficient (cusparse might be available by enabling as a tpl, or through kokkos-kernels or both); would something like:

```c++
#if defined(KOKKOS_ENABLE_CUDA) && (defined(TPL_ENABLE_CUSPARSE) || defined(KOKKOSKERNELS_ENABLE_TPL_CUSPARSE))
```
cover either case?